lib/puppet/provider/package/pacman.rb

```diff
@@ -28,7 +27,10 @@ def self.yaourt?
   # Checks if a given name is a group
   def self.group?(name)
     begin
-      !pacman("-Sg", name).empty?
+      execpipe([command(:pacman), "-Sg"], failonfail=false) do |process|
```

Comment:
This can just be `execpipe([command(:pacman), '-Sg'], false)`.  `failonfail=false` sets a local called `failonfail` to false and then passes the result of the expression.  Same result with just `false`, only no local is set.